src/CADL.Extension/Emitter.Csharp/samples/CadlFirstTest/firstTest.cadl

```diff
@@ -13,13 +13,17 @@ enum SimpleEnum {
     Four: "4"
 }
 
-@doc("Extensible enum")
-enum ExtensibleEnum {
+@doc("Extensible enum Values")
```

Comment:
You may consider removing the support for the Sealed Enum as we don't plan to support it in August MVP.
I believe the implementation in CodeGen could not handle sealed enum correctly.  
@archerzz, could you confirm?

Reply:
Yes. I don't think we plan to support sealed enum.

Reply:
To make it clear, @archerzz , @chunyu3 
fixed enums of strings are in-scope for August MVP.
fixed enums of ints are out-of-scope for August MVP.